packages/block-library/src/tag-cloud/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
```

Comment:
You don't have to include the editor.scss in index.js. There is now a editor.scss file in `/packages/block-libary/src/` where you can import the file.

Reply:
My bad this should be fixed now.